src/utils/api.js

```diff
@@ -4,10 +4,22 @@ const gamesAPI = axios.create({
   baseURL: "https://rk-board-games.herokuapp.com/api",
 });
 
-const getReviews = () => {
-  return gamesAPI.get("/reviews").then(({ data }) => {
+const getReviews = (category) => {
+  if (category === null) {
+    return gamesAPI.get("/reviews").then(({ data }) => {
+      return data;
+    });
+  } else {
+    return gamesAPI.get(`/reviews?category=${category}`).then(({ data }) => {
+      return data;
```

Comment:
we can skip the need for if conditions here by using the axios request params object which will handle queries for us - have a Google and try implementing it! It will help us out later too once we get to doing more advanced queries on our reviews endpoint